### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by rohmulan**
Update readme with link to GSOC proposal for the themeroller

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,5 @@
 This is the new repo for the Chassis ThemeRoller tool.
+
+The themeroller will  allow designing and editing themes by customizing existing [css-chassis](http://css-chassis.com) components and is currently under development. To know more, visit this proposal:
```
Comment (arthurvr) [inline README.md]: two spaces between the words `will` and `allow`

File: README.md [hunk 2]
```diff
@@ -1 +1,5 @@
...
+https://docs.google.com/document/d/1tKXT5_dDL6P9WHbmChu368pyyty8BV4nNQ4tFvoCcFI/edit?usp=sharing
+
```
Comment (arthurvr) [inline README.md]: let's remove this blank line.

File: README.md [hunk 3]
```diff
@@ -1 +1,5 @@
 This is the new repo for the Chassis ThemeRoller tool.
+
+The themeroller will  allow designing and editing themes by customizing existing [css-chassis](http://css-chassis.com) components and is currently under development. To know more, visit this proposal:
+https://docs.google.com/document/d/1tKXT5_dDL6P9WHbmChu368pyyty8BV4nNQ4tFvoCcFI/edit?usp=sharing
```
Comment (arthurvr) [inline README.md]: Let's not paste in this url but make a link.

File: README.md (hunk 1 above)
Comment (arthurvr) [inline README.md]: > css-chassis

do we want to write css chassis like that or like `CSS Chassis`? @sfrisk?

Comment (sfrisk) [inline README.md]: I often refer to it as the proper noun: Chassis, which is how the logo refers to it and the documentation at css-chassis.com.

File: README.md (hunk 3 above)
Comment (sfrisk) [inline README.md]: Agreed

**Comment by sfrisk** [comment]
Once the requested changes have been made on this, I'll pull this in.

**Comment by rohmulan** [comment]
I updated the PR @sfrisk

**Pull request closed without merging**